src/Chainweb/CutDB.hs

```diff
@@ -395,8 +395,10 @@ startCutDb
     -> RocksDbCas CutHashes
     -> IO (CutDb cas)
 startCutDb config logfun headerStore payloadStore cutHashesStore = mask_ $ do
-    logg Debug "obtain initial cut"
+    logg Info "obtain initial cut"
     initialCut <- readInitialCut
+    -- TODO before merge: is this a good idea or not?
+    -- deleteRangeRocksDb (_getRocksDbCas cutHashesStore) (Just $ over _1 succ $ casKey $ cutToCutHashes Nothing initialCut, Nothing)
```

Comment:
What are we planning to do here? I think, deleting would make sense, but not 100% sure.

Reply:
I think deleting makes sense, because in practice this feature exists to recover from broken forks, and in that case we certainly don't want the old cut hanging around until the new fork becomes heavier. I'll uncomment this.